src/main/resources/update/gitbucket-core_4.21.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <changeSet>
-  <createTable tableName="RELEASE_TAG">
+  <createTable tableName="RELEASE">
```

Comment:
does it mean that you _revert_ the change?

Reply:
Yes in 4.21.0. I modified migration to rename this table to `RELEASE_TAG` in 4.23.